modules/storage-azure-file-create-storage-class.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent-storage-azure-file.adoc
+
+[id="storage-create-azure-file-storage-class_{context}"]
+= Creating the Azure storage class
+
+StorageClasses are used to differentiate and delineate storage levels and
+usages. By defining a storage class, users can obtain dynamically provisioned
+persistent volumes.
+
+.Additional References
+
+* https://kubernetes.io/docs/concepts/storage/storage-classes/#azure-file[Azure File Storage Class]
+
+.Procedure
+
+. In the {product-title} console, click *Storage* -> *Storage Classes*.
+
+. In the storage class overview, click *Create Storage Class*.
+
+. Define the desired options on the page that appears.
+
+.. Enter a name to reference the storage class.
+
+.. Enter an optional description.
+
+.. Select the reclaim policy.
+
+.. Select `kubernetes.io/azure-file` from the drop down list.
```

Comment:
~drop down~

The example for drop-down lists in the style guide omits "drop-down."
> On the Background page, select a background from the list.

If you *must* include it for clarity, it's hyphenated:
> ! drop-down adj
Use only if you must specify the type of menu or list.